extensions/wikia/ContentReview/tests/ContentReviewHelperTest.php

```diff
@@ -8,61 +8,119 @@ public function setUp() {
 	}
 
 	/**
-	 * @dataProvider replaceWithLastApprovedRevisionProvider
+	 * @param bool $isJsPage
+	 * @param bool $inNamespace
+	 * @param string $contentType
+	 * @param bool $expected
+	 * @dataProvider shouldPageContentBeReplacedData
+	 */
+	public function testShouldPageContentBeReplaced( $isJsPage, $inNamespace, $contentType, $expected ) {
```

Comment:
I'd suggest using the standard naming convention of `test<method name>` so `testIsPageReviewed`, as that helper method may be used for other purposes.